test/modules/packages/distributedIters/checkDistributedIters.compopts

```diff
@@ -0,0 +1 @@
+--fast -I.
```

Comment:
Sounds good! I recall needing -I. in the beginning of the summer, but you're right that we don't appear to need it now.